client/modules/IDE/components/AssetSize.stories.jsx

```diff
@@ -0,0 +1,12 @@
+import AssetSize from './AssetSize';
+
+export default {
+  title: 'IDE/AssetSize',
+  component: AssetSize
+};
+
+export const Default = {
+  args: {
+    totalSize: 123
```

Comment:
This story shows up with a control for the `totalSize`, but changing it doesn't do anything because the default export gets the size from Redux.

Once we get the styles working and once we get this using the `args` then it might be cool to have stories for empty vs. full vs. somewhere in the middle.